lib/Controller/OpenProjectAPIController.php

```diff
@@ -197,15 +198,17 @@ public function linkWorkPackageToFile(int $workpackageId, int $fileId, string $f
 
 		try {
 			$result = $this->openprojectAPIService->linkWorkPackageToFile(
-				$workpackageId,
-				$fileId,
-				$fileName,
+				$values,
 				$this->userId,
 			);
 		} catch (OpenprojectErrorException $e) {
 			return new DataResponse($e->getMessage(), Http::STATUS_BAD_REQUEST);
```

Comment:
Maybe we can do it like this since we're throwing the same status code?
```suggestion
		} catch (OpenprojectErrorException | InvalidArgumentException $e) {
			return new DataResponse($e->getMessage(), Http::STATUS_BAD_REQUEST);
```